IDE/MonoDevelop/MonoDevelop.MonoGame.Mac/templates/MonoGameOSXProject.xpt.xml

```diff
@@ -27,6 +27,9 @@
 				<Reference type="Gac" refto="XamMac" SpecificVersion="false" />
 				<Reference type="Package" refto="MonoGame.Framework" />
 			</References>
+			<Packages>
+				<Package id="MonoGame.Framework.MacOS" version="3.3.0" targetFramework="net40" />
```

Comment:
Yes, Ieadly I want to be able to use the Nuget's we build during the build process. Is that possible now ? Do the Nugets get built before the Addin/Installer? Are they available as archives?